src/main/scala/mesosphere/marathon/upgrade/TaskReplaceActor.scala

```diff
@@ -125,6 +148,14 @@ class TaskReplaceActor(
     checkFinished()
   }
 
+  def checkForTasksToKill(): Int = {
```

Comment:
I'd name this killOldTasks, because it actually requests the old tasks to be killed.